### PR TITLE
added YYYY-MM-DD regex to date annotations

### PR DESCRIPTION
**Pull request by gkowalski**

### CODE REVIEW

File: server/openapi_server/controllers/text_date_annotation_controller.py [hunk 1]
```diff
@@ -33,6 +36,11 @@ def create_text_date_annotations():  # noqa: E501
                 "(-)(19[0-9][0-9]|20[0-9][0-9])", note._text)
             add_date_annotation(annotations, matches, "MM-DD-YYYY")
 
+            matches = re.finditer(
+                "(19[0-9][0-9]|20[0-9][0-9])-([0-1][0-9])-([0-9][0-9])",
```
Comment (tschaffter) [inline server/openapi_server/controllers/text_date_annotation_controller.py:40]: @gkowalski Why is the regex for "YYYY-MM-DD" smaller than the regex of "MM-DD-YYYY"?

Comment (gkowalski) [reply server/openapi_server/controllers/text_date_annotation_controller.py:40]: Better engineered, more concise.

Comment (tschaffter) [reply server/openapi_server/controllers/text_date_annotation_controller.py:40]: What does that mean? Can you apply your "better engineering" to the other regex?

Comment (tschaffter) [reply server/openapi_server/controllers/text_date_annotation_controller.py:40]: Your regex does not catch `1912-1-1`. This PR has been opened for quite some time now so I'll go ahead and close it.

**Comment by tschaffter** [comment]
Replaced by #79

**Pull request closed without merging**